nncf/common/quantization/quantizer_setup.py

```diff
@@ -68,6 +103,18 @@ def __str__(self):
     def __hash__(self):
         return hash(str(self))
 
+    def get_state(self) -> Dict[str, Any]:
+        """
+        Returns a dictionary with Python data structures (dict, list, tuple, str, int, float, True, False, None) that
+        represents state of the object.
+
+        :return: state of the object
+        """
+        return {
+            self._state_names.TARGET_NODE_NAME: self.target_node_name,
+            self._state_names.INPUT_PORT_ID: self.input_port_id
+        }
+
```

Comment:
Had to change builder state according to the changes in develop.
Previously, it was PTTargetPoint, now it's `ActivationQuantizationInsertionPoint` and `WeightQuantizationInsertionPoint`.
SOTA checkpoints were updated, locally eval is OK. build in CI is WIP